disnake/interactions/base.py

```diff
@@ -858,6 +861,9 @@ async def send_message(
             before deleting the message we just sent. If the deletion fails,
             then it is silently ignored.
 
+            .. versionchanged:: 2.7
+                Added support for ephemeral messages.
```

Comment:
note that in the case of an ephemeral message, they can only be deleted up to a maximum of 15 minutes.

Reply:
https://github.com/DisnakeDev/disnake/pull/816/commits/d9324ce869847f5ad94967c06aebf5f27a6bb96c